packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -699,25 +825,58 @@ export class StatsAnalyzer extends EventsScope {
             `StatsAnalyzer:index#compareLastStatsResult --> No share RTP packets sent`,
             currentStats.totalPacketsSent
           );
-        } else {
-          if (
-            currentStats.framesEncoded === previousStats.framesEncoded ||
-            currentStats.framesEncoded === 0
-          ) {
-            LoggerProxy.logger.info(
-              `StatsAnalyzer:index#compareLastStatsResult --> No share frames getting encoded`,
-              currentStats.framesEncoded
+        } else if (
+          currentStats.framesEncoded === previousStats.framesEncoded ||
```

Comment:
does this work? do we ever get to here if share frames are not encoded at all? aren't we going to fall into the condition for `totalPacketsSent` above?